.circleci/config.yml

```diff
@@ -6,17 +6,69 @@ jobs:
 
   build-python36:
     docker:
-      - image: ocrd/core
+      - image: ubuntu:18.04
```

Comment:
Why not `ocrd/core` as base image anymore?

Now we have to repeat many steps from core's Dockerfile here, and I don't like the idea of having to rebase `FROM ubuntu:18.04` in each and every module in the future...

Reply:
Main reason was the logging configuration in the Docker image which interferes with the testing (there was a file `ocrd.log` that was falsely listed as a resource of ocrd-tesserocr-recognize). We should debug this and change it back to `ocrd/core`, I was mostly trying to get the multi-version testing working with lots of trial and error.

Reply:
It should be possible to switch back to `ocrd/core` now, right?

Reply:
It's also urgent, because with the recent changes in core, we cannot just do `pip install ocrd` anymore:
```
FileNotFoundError: [Errno 2] No such file or directory: 'identify': 'identify'
```

Reply:
resolved in https://github.com/OCR-D/ocrd_tesserocr/pull/176/commits/f01cd274b0bd965f8d12cf35111d6d080fc09014